libs/wire-api/src/Wire/API/Routes/Public/Spar.hs

```diff
@@ -164,6 +167,10 @@ sparResponseURI Nothing =
 sparResponseURI (Just tid) =
   SAML.getSsoURI' (Proxy @APISSO) (Proxy @APIAuthResp) tid
 
+instance HasLink (Until 'V8 :> api) where
```

Comment:
Not so sure about this. We need this instance because of `getSsoURI` (https://hoogle.zinfra.io/file/nix/store/kjcia6s20r3lw0nvl1h68mba1d8m0nwy-saml2-web-sso-0.20-doc/share/doc/saml2-web-sso-0.20/html/SAML2-WebSSO-SP.html#v:getSsoURI). But I don't know yet if this works. In any case, it should probably go into the Version module.

Reply:
@fisx can we check this together?